packages/facade/src/contracts/hub.ts

```diff
@@ -222,7 +239,7 @@ class LidoTerraHub extends LidoTerraBaseContract {
       withdrawable_unbonded: { address },
     });
     return {
-      withdrawableUnbonded: new Int(response.withdrawable_unbonded),
+      withdrawableUnbonded: new Int(response.withdrawable),
```

Comment:
line does not follow "code convention" where model's field name in camel case = response field name in snake case

Reply:
renamed 👍